.github/workflows/test-stories.yaml

```diff
@@ -109,17 +109,18 @@ jobs:
       - uses: ./.github/actions/setup
       - name: Build Storybook for Chromatic
         # We want both stories and docs for the branch preview
-        run: pnpm turbo:storybook:build:chromatic-turbo-snap
+        run: pnpm turbo @docs/storybook#build:chromatic
       - id: publishChromatic
         name: Publish to Chromatic
         uses: chromaui/action@v11
         with:
           token: ${{ github.token }}
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-          storybookBuildDir: "./storybook/public"
+          storybookBuildDir: "./docs/storybook-static"
           onlyChanged: "!(main)"
           externals: |
             **/!(*.module).scss
+            **/!(*.module).css
             packages/components/**/*.css
```

Comment:
You'll want to take this out then, because it will match `.module.css` files within KAIO :)